aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -101,7 +101,7 @@ inline void __attribute__((always_inline)) QuantizeAvx2(
     int len,
     float inverse_scale,
     int64_t zero_point) {
-#if defined(__AVX2__) && defined(__FMA__)
+#if defined(CPU_CAPABILITY_AVX2) && defined(__FMA__)
```

Comment:
Please remove `defined(__FMA__)` since `CAPABILITY_AVX2` implies that `fma` instruction is available
```suggestion
#if defined(CPU_CAPABILITY_AVX2)
```